pkg/gateway/gateway.proto

```diff
@@ -365,3 +372,47 @@ message DeleteMachineResponse {
   bool ok = 1;
   string err_msg = 2;
 }
+
+message Token {
+  uint32 id = 1;
+  string external_id = 2;
+  string key = 3;
+  bool active = 4;
+  bool reusable = 5;
+  optional uint32 workspace_id = 6;
+  string token_type = 8;
+  google.protobuf.Timestamp created_at = 9;
+  google.protobuf.Timestamp updated_at = 10;
+  bool disabled_by_cluster_admin = 11;
+}
+
+message ListTokensRequest {}
+
+message ListTokensResponse {
+  bool ok = 1;
+  string err_msg = 2;
+  repeated Token tokens = 3;
+}
+
+message CreateTokenRequest {}
+
+message CreateTokenResponse {
+  bool ok = 1;
+  string err_msg = 2;
+  Token token = 3;
+}
+
+message ToggleTokenRequest { string external_id = 1; }
```

Comment:
I think we want to use `id` or `token_id` for this. I don't think we typically expose the external_id as "external".